### ISSUE TITLE
[Bug] ETH network ERC20 transactions list parse error

### ISSUE THREAD

**Issue by LPX55** [shapeshift/web#829]
Self explanatory: https://streamable.com/nyzed0

edit: Nevermind, doesn't seem to be float related.

![image](https://user-images.githubusercontent.com/16395727/150238194-7b20d6d3-5482-4cc2-a5fc-c9ba61ebd826.png)

**Comment by LPX55** [comment]
Checked two wallets/addresses -- the perpetrator seems to be Uniswap V2 token transfers.

**Comment by 0xdef1cafe** [comment]
related to #786

**Comment by 0xean** [comment]
currently blocked by tx metadata that will help define the needed precision.

**Comment by 0xean** [comment]
@0xApotheosis - can this be unblocked at this point?

**Comment by 0xApotheosis** [comment]
@0xean I'll check it out.

Noting a few things here for reference and posterity:

A transaction with a confirmed `precision` issue that I'll debug against: https://etherscan.io/tx/0xae3be17af6e1eee8f954dfb2ee9d47e7ce54bf1f92b2b2693ff21a999e4f42ca

Current, parsed, output from the Chain Adapter on this transaction:

```ts
{
  address: '0xef4FD55584434903aa6FC9baa61eAcDDe5355DA0',
  blockHash: '0xfc4657b0476f53a0b2ffe2cff3cdef940236ea074f3320fabb3eb22b52c87818',
  blockHeight: 13908803,
  blockTime: 1640895747,
  caip2: 'eip155:1',
  chain: 'ethereum',
  confirmations: 369274,
  fee: undefined,
  status: 'confirmed',
  tradeDetails: undefined,
  transfers: [
    {
      caip19: 'eip155:1/slip44:60',
      from: '0x69887e2DA0722401fb3d5618848683C3e922cCdf',
      to: '0xef4FD55584434903aa6FC9baa61eAcDDe5355DA0',
      type: 'receive',
      value: '9986070517210000'
    }
  ],
  txid: '0xae3be17af6e1eee8f954dfb2ee9d47e7ce54bf1f92b2b2693ff21a999e4f42ca'
}
```

Blockbook `api/v2/tx` response:

```json
{
  "txid": "0xae3be17af6e1eee8f954dfb2ee9d47e7ce54bf1f92b2b2693ff21a999e4f42ca",
  "vin": [
    {
      "n": 0,
      "addresses": [
        "0x69887e2DA0722401fb3d5618848683C3e922cCdf"
      ],
      "isAddress": true
    }
  ],
  "vout": [
    {
      "value": "9986070517210000",
      "n": 0,
      "addresses": [
        "0xef4FD55584434903aa6FC9baa61eAcDDe5355DA0"
      ],
      "isAddress": true
    }
  ],
  "blockHash": "0xfc4657b0476f53a0b2ffe2cff3cdef940236ea074f3320fabb3eb22b52c87818",
  "blockHeight": 13908803,
  "confirmations": 368951,
  "blockTime": 1640895747,
  "value": "9986070517210000",
  "fees": "2570024160249000",
  "ethereumSpecific": {
    "status": 1,
    "nonce": 1,
    "gasLimit": 21000,
    "gasUsed": 21000,
    "gasPrice": "122382102869",
    "data": "0x"
  }
}
```

**Comment by 0xApotheosis** [comment]
I mocked the identified transactions locally and found no issues. Everything is rendering correctly in both the "recent transactions" and "transaction history" components, and the transaction links work as expected.

I've checked in with @LPX55 and confirmed that the issue is no longer occurring with their wallet in production, either. 

Since this issue was reported, we have refactored the components the related logic sits in, and so its possible it was inadvertently, or deliberately, fixed at some point, or it was a one-off issue with @LPX55's local environment - we may never know 🤷‍♂️

<img width="618" alt="Screen Shot 2022-02-27 at 5 48 01 pm" src="https://user-images.githubusercontent.com/97164662/155871699-3d546b1d-7360-45f8-a4a5-ad6ae66f6b24.png">

<img width="331" alt="Screen Shot 2022-02-27 at 5 47 44 pm" src="https://user-images.githubusercontent.com/97164662/155871697-3cca6dea-da8e-4c9d-89f9-9d3e1bbe9aac.png">

Closing this out, as this is a non-issue.
